apps/gp2-server/src/data-providers/user.data-provider.ts

```diff
@@ -213,6 +199,45 @@ export const parseGraphQLUserToDataObject = ({
         institution,
       };
     }) || [];
+
+  const projects =
+    projectItems?.map(({ id: projectId, flatData: project }) => {
+      if (!project.status) {
+        throw new TypeError('Status not defined');
+      }
+      return {
+        id: projectId,
+        status: project.status,
+        title: project.title || '',
+        members:
+          project.members?.map((member) => {
+            const user = member.user && member.user[0];
```

Comment:
this is weird, it that thing from squidex were we can add multiple members to the same role?